packages/react-scripts/config/webpack.config.prod.js

```diff
@@ -272,8 +272,11 @@ module.exports = {
                 // We need to use our own loader until `babel-loader` supports
                 // customization
                 // https://github.com/babel/babel-loader/pull/687
```

Comment:
This comment needs deleted.

```
// We need to use our own loader until `babel-loader` supports
// customization
// https://github.com/babel/babel-loader/pull/687
```

Reply:
Thanks! Removed in 232e8927546b93bf97482c7e6f29c545c7b98185.